python/whylogs/api/store/sqlite_store.py

```diff
@@ -0,0 +1,65 @@
+import logging
+import os
+import sqlite3
+from datetime import datetime
+from typing import Optional, Union
+
+from whylogs.api.store import ProfileStore
+from whylogs.api.store.query import DateQuery, ProfileNameQuery
+from whylogs.core import DatasetProfile, DatasetProfileView
+
+logger = logging.getLogger(__name__)
+
+
+class SQLiteStore(ProfileStore):
+    def __init__(self, connection: Optional[sqlite3.Connection] = None):
+        self._db_location = os.getenv("SQLITE_STORE_LOCATION") or ":memory:"
+        self.conn = connection or sqlite3.connect(database=self._db_location)
+        self.cur = self.conn.cursor()
+
+    def list(self):
+        response = self.cur.execute("SELECT name FROM sqlite_schema WHERE type='table' ORDER BY name")
+        return [item[0] for item in response.fetchall()]
+
+    def get(self, query: Union[ProfileNameQuery, DateQuery]):
+        available_profiles = self.list()
+        if query.profile_name not in available_profiles:
+            logger.error(f"Profile {query.profile_name} does not exist!")
+            raise ValueError
+        if isinstance(query, ProfileNameQuery):
+            response = self.cur.execute(f"SELECT profile FROM {query.profile_name}")
+            profile_view = DatasetProfile().view()
+        elif isinstance(query, DateQuery):
+            sql_query = f"SELECT profile FROM '{query.profile_name}' WHERE date BETWEEN '{query.start_date}' AND '{query.end_date}';"
+            response = self.cur.execute(sql_query)
+        profile_view = DatasetProfile().view()
+        for item in response.fetchall():
+            profile_view.merge(DatasetProfileView.deserialize(item[0]))
+        return profile_view
+
+    def _insert_blob(self, profile_view: bytes, profile_name: str):
+        try:
+            profile_date = datetime.utcnow()
+            data_tuple = (profile_date, profile_view)
+            query = f"INSERT INTO {profile_name} (date, profile) VALUES (?, ?);"
```

Comment:
Should switch this over to "prepared queries" that we can reuse rather than doing string sub